docs/resources/dns_auth_zone.md

```diff
@@ -13,6 +13,14 @@ Manages an authoritative zone.
 ## Example Usage
 
 ```terraform
+resource "bloxone_keys_tsig" "test" {
```

Comment:
Don't use "test" as name in the examples

Reply:
Addressed